assets-management/services/svc-asset-delta-engine/test-data/gcp-all.json

```diff
@@ -3,7 +3,7 @@
         {
             "messageId": "059f36b4-87a3-44ab-83d2-661975830a7d",
             "receiptHandle": "AQEBwJnKyrHigUMZj6rYigCgxlaS3SLy0a...",
-            "body": "--tenant-id=saasdev --data-source=gcp --s3-path=mapper-results/saasdev/gcp/20240930-174929",
+            "body": "--tenant_id=98c28482-9bae-46bd-bd4e-58fa132e72c0 --data_source=gcp --s3_path=mapper-results/98c28482-9bae-46bd-bd4e-58fa132e72c0/gcp/20241108-040600 --skip_asset_count=true",
```

Comment:
I suggest that in the future, we throw test files that contain actual values to the ivan repo. Although, this is not sensitive data, we should clean it up in code repos to dummy values when checking in.

Reply:
Well, that's a brilliant idea. I wish I'd thought of it.